libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFSecurityLockout.m

```diff
@@ -215,23 +215,21 @@ + (void)validateTimer
     }
 }
 
-+ (void)setInactivityConfiguration:(NSUInteger)newPasscodeLength lockoutTime:(NSUInteger)newLockoutTime biometricAllowed:(BOOL)newBiometricAllowed
-{
-    // No Passcode
-    if (newLockoutTime == 0) {
-        if (securityLockoutTime == 0) {
-            [SFSecurityLockout unlockSuccessPostProcessing:SFSecurityLockoutActionNone];
-        } else {
-            // '0' is a special case.  We can't turn off passcodes unless all other users' passcode policies
-            // are also off.
-            if (![SFSecurityLockout nonCurrentUsersHavePasscodePolicy]) {
-                [SFSecurityLockout clearAllPasscodeState];
-                [SFSecurityLockout unlock:YES action:SFSecurityLockoutActionPasscodeRemoved];
-                return;
-            }
-        }
++ (BOOL)needsPasscodeFlow:(NSUInteger)newLockoutTime {
+    return newLockoutTime !=0 || [self usersHavePasscodePolicy];
```

Comment:
Should this be `newLockoutTime` or `securityLockoutTime`?

Reply:
Should be the newValue received from a login flow.